kytos/utils/config.py

```diff
@@ -82,22 +82,15 @@ def set_env_or_defaults(self):
         elif not self.config.has_option('napps', 'repo'):
             self.config.set('napps', 'repo', 'https://napps.kytos.io/repo/')
 
-        self._set_napps_path(napps_path)
-
-    def _set_napps_path(self, napps_path):
-        """Set paths if NAPPS_PATH is given or if not found in config."""
-        if napps_path or not self.config.has_option('napps', 'enabled_path'):
-            if not napps_path:  # default paths
-                base = os.environ.get('VIRTUAL_ENV') or '/'
-                napps_path = os.path.join(base, 'var', 'lib', 'kytos', 'napps')
-            self.config.set('napps', 'enabled_path', napps_path)
-            self.config.set('napps', 'installed_path',
-                            os.path.join(napps_path, '.installed'))
+        if kytos_server is not None:
+            self.config.set('kytos', 'api', kytos_server)
+        elif not self.config.has_option('kytos', 'api'):
+            self.config.set('kytos', 'api', 'http://localhost:8181/')
```

Comment:
We can make this code more pythonic and more clear doing:
```python
from collections import namedtuple

Option = namedtuple('Option', ['section', 'name', 'env_var', 'default_value'])
options = [Option('auth', 'user', 'NAPPS_USER', None),
               Option('auth', 'token', 'NAPPS_TOKEN', None),
               Option('napps', 'api', 'NAPPS_API_URI', 'https://napps.kytos.io/api/'),
               Option('napps', 'repo', 'NAPPS_REPO_URI', 'https://napps.kytos.io/repo'),
               Option('kytos', 'api', 'KYTOS_API', 'http://localhost:8181')]

for option in options:
    env_value = os.environ.get(option.env_var)
    if env_value:
        self.config.set(option.section, option.name, env_value)
    elif option.default_value:
        self.config.set(option.section, option.name, option.default_value)
```
This avoids code repetition and therefore mistakes, while facilitation the addition of new parameters.